images/linux/scripts/installers/graalvm.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+source $HELPER_SCRIPTS/install.sh
+
+#Install GraalVM
+tmp_graalvm=/tmp/graalvm
+export JAVA_HOME_GRAALVM_11=/usr/local/graalvm/graalvm-ce-java11*
+export GRAALVM_11_BIN=/usr/local/graalvm/graalvm-ce-java11*/bin
+
+mkdir -p $tmp_graalvm
+url=$(curl -s https://api.github.com/repos/graalvm/graalvm-ce-builds/releases/latest | jq -r '.assets[].browser_download_url | select(contains("graalvm-ce-java11-linux-amd64"))')
+download_with_retries "$url" "$tmp_graalvm" "graalvm-archive.tar.gz"
+mkdir /usr/local/graalvm
```

Comment:
create additional variable `GRAALVM_ROOT` because you have a lot of duplicates `/usr/local/graalvm`

Reply:
Created it